package.json

```diff
@@ -2,14 +2,36 @@
   "name": "apostrophe-storybook",
   "version": "1.0.0",
   "description": "",
+  "author": "Apostrophe Technologies, Inc.",
+  "scripts": {
+    "test:unit": "vue-cli-service test:unit",
+    "build-storybook": "build-storybook -s ./public -o .out",
+    "deploy-storybook": "npm run build-storybook && storybook-to-ghpages -- --out=.out && rm -rf .out",
+    "dev": "start-storybook -s ./public -p 9001 -c .storybook",
+    "storybook": "start-storybook -s ./public -p 9001 -c .storybook",
+    "test": "npx eslint . --ext .js,.vue"
+  },
   "main": "index.js",
   "dependencies": {
+    "vue": "^2.6.11",
+    "vue-loader": "^15.9.1",
+    "vue-material-design-icons": "^3.4.0",
+    "vue-template-compiler": "^2.6.11",
+    "vuedraggable": "^2.23.2"
+  },
+  "devDependencies": {
     "@babel/core": "^7.8.7",
     "@storybook/addon-a11y": "^5.3.17",
     "@storybook/addon-contexts": "^5.3.17",
     "@storybook/addon-knobs": "^5.3.17",
     "@storybook/addon-storysource": "^5.3.17",
+    "@storybook/storybook-deployer": "^2.8.6",
     "@storybook/vue": "^5.3.17",
```

Comment:
@gregvanbrug Are you sure this isn't going to break the deployment since these are now dev dependencies?

Reply:
We're running deploys locally so there shouldn't be an issue. All the @storybook modules are moved to dev dependencies so they don't install when this module is installed elsewhere as we'll only need the components not storybook itself.